packages/grafana-data/src/utils/csv.ts

```diff
@@ -289,7 +290,7 @@ export function toCSV(data: DataFrame[], config?: CSVConfig): string {
         if (i > 0) {
           csv += config.delimiter;
         }
-        csv += fields[i].name;
+        csv += `"${getFieldDisplayName(fields[i], series).replace('"', '"""')}"`;
```

Comment:
@peterholmberg this will replace only first occurence of `"`, think you need to pass a regex

Reply:
Any pointers on that? I'm fumbling in the dark here.

Reply:
`replace(/"/g, '"""');`

should do the job

Reply:
I don't know, still haven't found a solution that works for LibreOffice


![Screenshot from 2020-05-14 14-49-38](https://user-images.githubusercontent.com/946275/81936209-2bf01500-95f2-11ea-9ab2-6c4570bd500f.png)

Reply:
I think this is fine for now, we cannot coverall csv parsers without an option for escape method

Reply:
Yeah, would be good to test the output agains Excel. MacOs Numbers has the same problem ^^

Reply:
Strange using double quotes to escape worked for me in numbers

Reply:
The replace is wrong. Try replace with only two double quotes

Reply:
You are right @torkelo, I haven't noticed that the change wasn't pushed

Reply:
pushed now

Reply:
Works in Excel too (O365)